test/rekt/sink_binding_test.go

```diff
@@ -45,6 +46,22 @@ func TestSinkBindingV1Deployment(t *testing.T) {
 	env.Test(ctx, t, sinkbinding.SinkBindingV1Deployment(ctx))
 }
 
+func TestSinkBindingV1DeploymentTLS(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+		environment.Managed(t),
+		eventshub.WithTLS(t),
+	)
+	t.Cleanup(env.Finish)
```

Comment:
Nit: The cleanup is already ensured by calling `environment.Managed(t)`.  The `t.Cleanup(env.Finish)` is called inside it.